cirq-core/cirq/circuits/qasm_output_test.py

```diff
@@ -590,3 +590,30 @@ def test_reset():
 reset q[1];
     """.strip()
     )
+
+
+def test_different_sized_registers():
+    qubits = cirq.LineQubit.range(2)
+    c = cirq.Circuit(cirq.measure(qubits[0], key='c'), cirq.measure(qubits, key='c'))
+    output = cirq.QasmOutput(
+        c.all_operations(), tuple(sorted(c.all_qubits())), header='Generated from Cirq!'
+    )
+    assert (
+        str(output).strip()
```

Comment:
we can as well test a dressed output -
```suggestion
        str(output)
```